deepface/DeepFace.py

```diff
@@ -91,15 +92,15 @@ def build_model(model_name):
 
 
 def verify(
-    img1_path,
-    img2_path,
-    model_name="VGG-Face",
-    detector_backend="opencv",
-    distance_metric="cosine",
-    enforce_detection=True,
-    align=True,
-    normalization="base",
-):
+    img1_path: str,
```

Comment:
all image path variables (img_path, img1_path, img2_path) can be string or numpy array. So, these should accept numpy array as

```python
from typing import Union
import numpy as np

def verify(img1_path: Union[str, np.ndarray]):
   pass
```